### ISSUE TITLE
paper-select broken with custom search action without options

### ISSUE THREAD

**Issue by wahhaj** [miguelcobain/ember-paper#1025]
Ember Power Select allows you to [provide a `search` action instead of `options`](https://ember-power-select.com/docs/custom-search-action) but this functionality doesn't seem to work with `paper-select` anymore. It used to work up until `1.0.0-beta.7`.

e.g. on the Select page in the demo app, with this `action`

```js
searchVegetables(value) {
  return this.vegetables.filter(({ name }) => name.includes(value))
},
```

and this template 
```hbs
  {{#paper-select
    placeholder="Vegetable"
    selected=selectedVegetable
    search=(action 'searchVegetables')
    searchPlaceholder="Search for a vegetable.."
    searchEnabled=true
    onChange=(action (mut selectedVegetable))
    as |vegetable|
  }}
    {{vegetable.name}}
  {{/paper-select}}
```

clicking on the Select component shows these errors in the console

![screen shot 2018-11-19 at 1 23 16 pm](https://user-images.githubusercontent.com/1515099/48726783-67c53c80-ebfe-11e8-93de-25c945d28dad.png)

and the whole app becomes unresponsive, creating further instances of the same error anytime you click anywhere.

**Comment by miguelcobain** [comment]
I guess #1027 fixed this, but there still some problem with the searchable paper-select, at least on the docs (dummy app).

Basically I can open the select, filter options, etc, but clicking one option doesn't select it.
Can you reproduce it?

**Comment by wahhaj** [comment]
@miguelcobain Yeah I can reproduce that, but it's not a problem with the `paper-select` component itself - the example searchable paper-select in the dummy app has a mistake.

Instead of `value=selectedVegetable`, it should be `selected=selectedVegetable`.
And `closeOnSelect=false` makes it look feel like nothing was selected when you click on an item.

I've created a PR #1030 to fix this issue.

**Comment by miguelcobain** [comment]
@wahhaj Thanks so much!
Closing this as it is unrelated.
